src/quadratic.jl

```diff
@@ -0,0 +1,66 @@
+"""
+Helper function for quadratic_interval
+"""
+function quadratic_helper(a::Number, b::Number, c::Number, L::AbstractArray)
+
+    Δ = b*b - 4*a*c
+
+    Δ < 0 && return
+
+    Δ = sqrt(Δ)
+
+    if (b >= 0)
+        x0 = -0.5 * (b + Δ)
+
+    else
+        x0 = -0.5 * (b - Δ)
```

Comment:
Why is this distinction necessary?

Reply:
According to the Hansen book (Page 181), this procedure leads to minimisation of rounding errors.